gemfiles/rails3.2.gemfile

```diff
@@ -3,3 +3,4 @@ source "https://rubygems.org"
 gemspec :path=>"../"
 
 gem "actionmailer", "~> 3.2.18"
+gem "test-unit"
```

Comment:
yeah, adding  `test-unit-minitest` fixes it too and does not add a test-unit ... basically it's the minitest part that got removed from 2.2